src/Core/OAuth/OAuth2/OAuth2LoginHelper.php

```diff
@@ -259,14 +269,50 @@ public function refreshToken(){
        }
     }
 
+    /**
```

Comment:
@alexeycarpe I believe these changes should go one level down into the file: `src/Core/HttpClients/CurlHttpClient.php`.

Reply:
also agree

Reply:
PR udated